src/ophys_etl/modules/suite2p_registration/utils.py

```diff
@@ -28,7 +28,11 @@ def projection_process(data: np.ndarray,
     else:
         raise ValueError("projection can be \"max\" or \"avg\" not "
                          f"{projection}")
-    proj = np.uint8(proj * 255.0 / proj.max())
+
+    mn = proj.min()
+    mx = proj.max()
+    delta = mx-mn
+    proj = np.round((proj.astype(float)-mn) * 255.0 / delta).astype(np.uint8)
```

Comment:
I don't think `astype(float)` is needed.

Also this is a general purpose algorithm to convert to uint8 that we use all over the place. Is there no common place for this?

Reply:
Fair point. We can just make a call to `ophys_etl.utils.array_utils.normalize_array`